### DISCUSSION TITLE
Nest10 support ?

### DISCUSSION THREAD

**Discussion by gterras** [GrapeoffJS/kindagoose#166] [Q&A] [answered]
Hi and thanks for the great lib! 

Do you plan to update it to Nest10 soon ? It looks like it's already compatible (working fine with `npm --force`) but it would be easier to update this package dependencies. 

Thanks !

**Comment by findingtheconcept** [comment, marked as answer]
Hi! Thanks to github notifications I didn't see this message. Yeah, I will update kindagoose soon.

**Reply by findingtheconcept** [reply]
Omg, too months ago... ahah

**Reply by gterras** [reply]
Thanks mate good to hear! For the record a quick workaround is : 

```
"overrides": {
	"kindagoose": {
		"@nestjs/common": "^10.1.2",
		"@nestjs/core": "^10.1.2"
	}
},  
```

**Comment by findingtheconcept** [comment]
Just published the newest version: https://github.com/GrapeoffJS/kindagoose/releases/tag/1.3.7

**Reply by findingtheconcept** [reply]
Seems it works as expected with NestV10
